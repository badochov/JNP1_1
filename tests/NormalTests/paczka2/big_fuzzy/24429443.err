Error in line 1: EsHwlFbI40 S41 74,2
Error in line 5: S41 S41 12,1
Error in line 17: R0HD1 A205 90,7
Error in line 9: S41 A690 57,4
Error in line 25: whRCJuCkgb A205 74,9
Error in line 26: whRCJuCkgb A690 93,3
Error in line 20: pX6KeNsi A244 399,6
Error in line 32: whRCJuCkgb A205 66,2
Error in line 38: whRCJuCkgb S41 -4,6
Error in line 35: whRCJuCkgb S41 57,3
Error in line 50: MrLfmN A205 57,3
Error in line 54: S41 A244 9,8
Error in line 73: whRCJuCkgb A205 94,1
Error in line 86: S41 A690 49,8
Error in line 90: whRCJuCkgb S41 3,2
Error in line 93: R0HD1 A205 51,7
Error in line 108: EsHwlFbI40 A244 84,0
Error in line 109: R0HD1 A205 0,0
Error in line 114: S41 A244 1,4
Error in line 141: S41 A205 221,04
Error in line 147: TMxNy92 A205 20,3
Error in line 149: TMxNy92 A244 82,6
Error in line 143: EsHwlFbI40 S41 45,7
Error in line 120: S41 A205 88,9
Error in line 158: S41 S41 71,2
Error in line 183: EsHwlFbI40 A244 46,73
Error in line 191: TMxNy92 A205 15,2
Error in line 190: EsHwlFbI40 A244 50,5
Error in line 189: MrLfmN A205 97,7
Error in line 204: EsHwlFbI40 A244 39,5
Error in line 210: R0HD1 S41 91,A
Error in line 207: R0HD1 A244 55,2
Error in line 197: whRCJuCkgb A690 50,6
Error in line 206: EsHwlFbI40 A205 5,7
Error in line 224: TMxNy92 S41 70,9
Error in line 225: MrLfmN A205 32,1
Error in line 235: TMxNy92 A205 53,8
