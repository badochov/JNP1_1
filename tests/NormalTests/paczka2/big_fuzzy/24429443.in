EsHwlFbI40 S41 74,2
EsHwlFbI40 A690 89,8
pX6KeNsi A244 15,5
EsHwlFbI40 A690 396,9
S41 S41 12,1
MrLfmN A205 48,7
pX6KeNsi A244 8,4
?A244
S41 A690 57,4
R0HD1 S41 71,1
MrLfmN A205 34,6

pX6KeNsi A244 86,8
?
R0HD1 S41 81,0
pX6KeNsi A244 86,3
R0HD1 A205 90,7
R0HD1 A244 26,6
R0HD1 A244 3,2
pX6KeNsi A244 399,6
S41 A244 92,2
R0HD1 S41 62,9
R0HD1 S41 248,5
S41 A244 72,9
whRCJuCkgb A205 74,9
whRCJuCkgb A690 93,3
MrLfmN A244 82,4

S41 A244 35,7
MrLfmN A244 54,6
MrLfmN A690 531,3
whRCJuCkgb A205 66,2
pX6KeNsi A690 28,0
MrLfmN A690 178,0
whRCJuCkgb S41 57,3
pX6KeNsi A690 50,1
S41 A244 98,6
whRCJuCkgb S41 -4,6
S41 A244 112,4
pX6KeNsi A244 23,4
R0HD1 A205 56,0
pX6KeNsi A244 113,5
EsHwlFbI40 A690 45,0
whRCJuCkgb A244 7,1
?S41
S41 A244 29,6
TMxNy92 A205 44,2
whRCJuCkgb A244 25,3
R0HD1 A205 234,6
MrLfmN A205 57,3
TMxNy92 A205 209,5
?pX6KeNsi
MrLfmN S41 46,0
S41 A244 9,8
R0HD1 A244 63,1
?MrLfmN
?S41
pX6KeNsi A244 17,2
MrLfmN S41 10,6
EsHwlFbI40 A690 158,8
pX6KeNsi A244 3,1
R0HD1 A244 50,5
S41 A690 61,2
TMxNy92 A244 37,7
?A690
whRCJuCkgb A690 7,3
R0HD1 A205 62,8
S41 A690 114,4
TMxNy92 A244 103,3
?A205
whRCJuCkgb A690 1,8
MrLfmN A205 61,0
whRCJuCkgb A205 94,1
whRCJuCkgb S41 91,5
?
MrLfmN A205 269,0
?A205
?
R0HD1 A205 207,9
whRCJuCkgb S41 23,0
?A690
?S41
pX6KeNsi A690 74,9
MrLfmN A690 51,1
pX6KeNsi A690 156,5
S41 A690 49,8
pX6KeNsi S41 45,5
S41 A205 12,6
S41 A205 2,5
whRCJuCkgb S41 3,2
MrLfmN A690 21,0
?
R0HD1 A205 51,7

EsHwlFbI40 A690 22,4
EsHwlFbI40 A690 5,4

pX6KeNsi S41 21,4
S41 A244 10,6
whRCJuCkgb A244 73,9
pX6KeNsi A205 17,8
?
MrLfmN A244 97,7
R0HD1 S41 23,3
whRCJuCkgb A244 87,2
R0HD1 S41 71,3
?TMxNy92
EsHwlFbI40 A244 84,0
R0HD1 A205 0,0
S41 A244 36,9
?A690
MrLfmN A244 96,7
EsHwlFbI40 S41 38,1
S41 A244 1,4
pX6KeNsi A205 37,0
R0HD1 A690 1,9
whRCJuCkgb A205 99,6
EsHwlFbI40 S41 148,6
R0HD1 A690 7,6
S41 A205 88,9
MrLfmN A244 6,7
?pX6KeNsi
?
EsHwlFbI40 A690 95,9
?A690
MrLfmN A244 30,4
whRCJuCkgb A205 136,5
?A244
EsHwlFbI40 A690 209,1
pX6KeNsi A244 0,2
MrLfmN A690 89,2
MrLfmN A690 384,5
pX6KeNsi A244 0,6
MrLfmN A690 95,7
MrLfmN A690 412,5
pX6KeNsi S41 59,8
whRCJuCkgb S41 76,3
?A244
TMxNy92 A244 30,0
?TMxNy92
S41 A205 221,04
TMxNy92 A244 114,3
EsHwlFbI40 S41 45,7
?MrLfmN
pX6KeNsi S41 278,7
pX6KeNsi A205 92,7
TMxNy92 A205 20,3
?S41
TMxNy92 A244 82,6
?A690
TMxNy92 S41 39,6
EsHwlFbI40 A244 14,9
pX6KeNsi A205 436,6
S41 S41 58,6
TMxNy92 S41 114,0
S41 S41 53,5
pX6KeNsi A690 1,3
S41 S41 71,2
EsHwlFbI40 A244 15,2
whRCJuCkgb S41 74,0
whRCJuCkgb A244 13,0
R0HD1 A205 9,2
TMxNy92 A690 77,3
R0HD1 A205 8,6
pX6KeNsi A690 6,0
?R0HD1
?S41
MrLfmN A205 34,8
TMxNy92 A690 319,2
pX6KeNsi S41 4,4
pX6KeNsi S41 4,0
MrLfmN A205 94,3
whRCJuCkgb A244 27,4
whRCJuCkgb A690 60,7
whRCJuCkgb A690 177,9
?A244
pX6KeNsi A244 30,7
pX6KeNsi A244 51,9
S41 A205 16,3
?
?
R0HD1 A205 82,3
EsHwlFbI40 A244 46,73
MrLfmN A244 17,4
S41 A205 49,2
S41 A205 99,8
R0HD1 A205 112,8
MrLfmN A244 54,1
MrLfmN A205 97,7
EsHwlFbI40 A244 50,5
TMxNy92 A205 15,2
TMxNy92 S41 92,6
S41 A205 339,3
S41 A244 16,1
S41 A244 14,2
EsHwlFbI40 A205 84,7
whRCJuCkgb A690 50,6
MrLfmN A690 58,4
MrLfmN A690 213,7
EsHwlFbI40 A205 398,1
S41 A244 65,4
pX6KeNsi S41 21,6
?A205
EsHwlFbI40 A244 39,5
TMxNy92 S41 23,2
EsHwlFbI40 A205 5,7
R0HD1 A244 55,2

pX6KeNsi S41 54,4
R0HD1 S41 91,A
S41 A244 172,0
R0HD1 A690 28,3
whRCJuCkgb A244 95,9

whRCJuCkgb A244 120,8
?S41
TMxNy92 S41 31,1
R0HD1 A690 114,3
whRCJuCkgb A244 2,8
S41 A690 42,6
TMxNy92 S41 89,3
EsHwlFbI40 A244 16,2
S41 A690 189,6
TMxNy92 S41 70,9
MrLfmN A205 32,1
TMxNy92 A690 0,9
S41 A244 30,4
EsHwlFbI40 A244 63,3
TMxNy92 A690 3,6
MrLfmN S41 36,6
R0HD1 A244 13,6
EsHwlFbI40 A205 58,0
?A244
whRCJuCkgb A244 8,5
TMxNy92 A205 53,8
whRCJuCkgb A690 51,8
EsHwlFbI40 A205 279,0
MrLfmN S41 2,9
TMxNy92 A690 0,7
TMxNy92 A690 3,5
whRCJuCkgb A690 116,0
EsHwlFbI40 A205 15,2
whRCJuCkgb S41 36,8
S41 A244 135,3
MrLfmN A244 42,1
S41 S41 63,5
MrLfmN A244 38,7

EsHwlFbI40 A205 50,8
MrLfmN S41 2,7