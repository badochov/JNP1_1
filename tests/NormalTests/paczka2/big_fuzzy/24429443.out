A244 7,1
EsHwlFbI40 A 307,1
MrLfmN A 14,1
pX6KeNsi A 7,1
A205 14,1
A244 7,1
A690 307,1
S41 A 82,2
S41 195,5
pX6KeNsi A 119,8
MrLfmN A 395,2
S41 A 165,0
S41 195,5
A690 796,3
A205 358,0
EsHwlFbI40 A 420,9
MrLfmN A 395,2 S 35,4
R0HD1 A 214,6 S 195,5
S41 A 218,2
TMxNy92 A 230,9
pX6KeNsi A 133,9
whRCJuCkgb A 23,7
S41 230,9
A205 358,0
A244 424,4
A690 855,0
A205 566,0
EsHwlFbI40 A 420,9
MrLfmN A 603,2 S 35,4
R0HD1 A 214,6 S 195,5
S41 A 218,2
TMxNy92 A 230,9
pX6KeNsi A 133,9
whRCJuCkgb A 23,7
S41 230,9
A205 566,0
A244 424,4
A690 855,0
A690 855,0
S41 A 218,2
S41 299,4
EsHwlFbI40 A 420,9
MrLfmN A 633,3 S 35,4
R0HD1 A 359,7 S 195,5
S41 A 228,3
TMxNy92 A 230,9
pX6KeNsi A 215,5
whRCJuCkgb A 23,7 S 68,5
S41 299,4
A205 721,2
A244 424,4
A690 966,7
EsHwlFbI40 A 437,9
MrLfmN A 633,3 S 35,4
R0HD1 A 359,7 S 195,5
S41 A 228,3
TMxNy92 A 230,9
pX6KeNsi A 215,5 S 24,1
whRCJuCkgb A 23,7 S 68,5
S41 323,5
A205 721,2
A244 424,4
A690 983,7
TMxNy92 A 230,9
A690 983,7
pX6KeNsi A 234,7 S 24,1
EsHwlFbI40 A 437,9 S 110,5
MrLfmN A 634,3 S 35,4
R0HD1 A 365,4 S 243,5
S41 A 254,6
TMxNy92 A 230,9
pX6KeNsi A 234,7 S 24,1
whRCJuCkgb A 37,0 S 68,5
S41 482,0
A205 740,4
A244 465,0
A690 989,4
A690 989,4
A244 488,7
A244 489,1
TMxNy92 A 230,9
MrLfmN A 1270,1 S 35,4
S41 A 254,6
S41 700,9
A690 1714,7
R0HD1 A 366,0 S 243,5
S41 A 254,6 S 5,1
S41 782,7
A244 588,1
EsHwlFbI40 A 551,4 S 110,5
MrLfmN A 1329,6 S 35,4
R0HD1 A 366,0 S 243,5
S41 A 254,6 S 5,1
TMxNy92 A 557,1 S 74,4
pX6KeNsi A 604,9 S 243,4
whRCJuCkgb A 205,5 S 70,8
S41 783,1
A205 1181,3
A244 609,3
A690 2078,5
EsHwlFbI40 A 551,4 S 110,5
MrLfmN A 1329,6 S 35,4
R0HD1 A 366,0 S 243,5
S41 A 254,6 S 5,1
TMxNy92 A 557,1 S 74,4
pX6KeNsi A 604,9 S 243,4
whRCJuCkgb A 205,5 S 70,8
S41 783,1
A205 1181,3
A244 609,3
A690 2078,5
A205 1797,6
S41 A 635,5 S 5,1
S41 885,3
A244 826,5
